_commands/basics/seq.md

```diff
@@ -0,0 +1,76 @@
+---
+---
+
+seq
+-------
+### Overview
```

Comment:
You can delete this (it shows up on the homepage and looks weird)
![image](https://cloud.githubusercontent.com/assets/5882053/14057694/a6948360-f2de-11e5-9c51-7c71d394aff1.png)